### ISSUE TITLE
Game launches, I can play world once, then when I exit to menu screen, I can't load up any world again, it just goes back into the menu screen.

### ISSUE THREAD

**Issue by Geekerandy** [MyM-ModpackTeam/GalacticScience#20]
So the issue is that I can load up any singleplayer world I've made and when I go back to the main menu screen, I can't go into a singleplayer world again, just sends me back to the main menu again. The only way to load up the world again is to relaunch the modpack. Then the same thing occurs again after I play one world, and then I can't log in to another world again. Note: will not even join the world I was just in as well. 
Here's the log (I played and restarted a few times, recommend going to entry 13,000 before looking.): http://paste.feed-the-beast.com/view/8438faff 
From what I could tell, the "state engine" is in a incorrect state.

Edit: Also I have used the FTB Launcher. Should I download the MyM Launcher and restart my whole progress for the case that it is updated and fixed?

Second Edit: Thank you in advance.

Third Edit: Here is a smaller, replicated version of the bug/error http://paste.feed-the-beast.com/view/54b5321f

**Comment by NolanSyKinsley** [comment]
I ran into this issue as well. The launcher is not at fault here. Modtweaker just needs to be updated to latest, but that means updating minetweaker, ex nihilo and ex astris to current as well. I have tested these updates very briefly and they seem to be working fine, but I already updated to latest forge and galacticraft myself so I don't know if these updates can be applied to the version of forge in the base modpack.

**Comment by Geekerandy** [comment]
@NolanSyKinsley Thank you for confirming it's not just my computer. I'm not a modding person (I just slap on mods and change a thing or two in the config), so your confirmation helps.
Thank you, and have a good day.

**Comment by Slind14** [comment]
There are too many mods relying on each other. We have spent hour on hours to find versions of each mod that all work together. All the mods of this set will probably never be updated as there is always one that doesn't work with one of the others. :/

**Comment by NolanSyKinsley** [comment]
@Slind14 I respectfully disagree. Most respectable mods have changed to rely on the forge infrastructure. Very few mods have failed to comply and thus exist outside the realm of supported functions and thus suffer consequences when people vary as they naturally do.

**Comment by NolanSyKinsley** [comment]
We just need to get enough people to put enough pressure on the dev to incorporate mine/modtweaker functionality into the mod. or if the source is available we could just take ont he project ourselves.

**Comment by Slind14** [comment]
First of all we need to find a new maintainer for Ex Astris (Rebirth).

**Comment by NolanSyKinsley** [comment]
lol, that was the mod I was referencing =P

**Comment by Geekerandy** [comment]
Well, seems like a complicated problem. I've experienced this same thing on another modpack but have forgotten what it was sinceit was last year that I last encountered it until now.

**Comment by drz2k** [comment]
I tried for hours with slind to get the mods updated and since we got crashes/nei not working/modtweaker not working we had to go back to this combination of mods.

Like he said, main problem is that some of the mods are allready without maintainers again.

If you happen to find an updated combination of the mods that work flawless, feel free to contribute and make a pull request. Until then, having to restart the client is a minor problem.
